packages/components/src/components/PipelineRun/PipelineRun.js

```diff
@@ -190,6 +239,11 @@ export /* istanbul ignore next */ class PipelineRunContainer extends Component {
       );
     }
     const taskRuns = this.loadTaskRuns(pipelineRun, taskRunNames);
+
+    if (taskRuns === []) {
+      return <div>No task runs found for this PipelineRun</div>;
```

Comment:
`TaskRuns`

also should be using react-intl here to externalise the string, either `<FormattedMessage ... />` or `intl.formatMessage(...)`